spec/cucumberConf.js

```diff
@@ -0,0 +1,35 @@
+var env = require('./environment');
```

Comment:
I have added a basic cucumber config file in `docs/frameworks.md` so that users can refer to it.